packages/components/switch/src/SwitchInput.styles.ts

```diff
@@ -60,20 +58,25 @@ export const thumbStyles = cva(
         sm: ['h-sz-20', 'w-sz-20'],
         md: ['h-sz-24', 'w-sz-24'],
       }),
+      checked: {
+        false: 'text-on-surface/dim-4',
+      },
     },
-    defaultVariants,
-  }
-)
-
-export const thumbCheckSVGStyles = cva(
-  ['transition-opacity duration-200', 'group-spark-state-unchecked:opacity-0 '],
-  {
-    variants: {
-      size: makeVariants<'size'>({
-        sm: ['h-sz-10 w-sz-10'],
-        md: ['h-sz-12 w-sz-12'],
-      }),
+    defaultVariants: {
+      size: 'sm',
+      checked: false,
     },
-    defaultVariants,
   }
 )
+
+export const thumbCheckSVGStyles = cva(['transition-opacity duration-200'], {
+  variants: {
+    size: makeVariants<'size'>({
+      sm: ['h-sz-10 w-sz-10'],
+      md: ['h-sz-12 w-sz-12'],
```

Comment:
Should the classes be separated by a comma ? you are doing it above

Reply:
```suggestion
      sm: ['h-sz-10', 'w-sz-10'],
      md: ['h-sz-12', 'w-sz-12'],
```